clar2wasm/src/lib.rs

```diff
@@ -29,6 +29,9 @@ pub const BLOCK_LIMIT_MAINNET_21: ExecutionCost = ExecutionCost {
     runtime: 5_000_000_000,
 };
 
+// prefix for standard library functions
+pub const STDLIB_PREFIX: &str = "stdlib";
```

Comment:
What is the point for this? We still call the functions exported from the Wasm by name, so why would we use `format!` all the time? That seems verbose, and adds a computation at each function call instead of just passing a constant `str`.
Or if we keep this for X reasons, how about a function `call_from_stdlib` that adds it automatically?

Reply:
@Acaccia I am not also satisfied with that amount of `STDLIB_PREFIX` spread in to the codebase. I'll work on that.

Does your main point is to avoid calling `format!`?

Thanks!

Reply:
That's kind of annoying that we have to remember to use `format!` now when we call a function, but my main point is that we are concatenating two compile-time known strings here. So I don't see the point making our life more complex and having this call because ultimately, we are just calling a function from Wasm that is called `stdlib.xxxx`, so why not just call it like this?

If one day we decide to change the prefix for whatever reason, obviously it makes sense to have this[^1], but in this case, I would still like to just call a function that will add the `stdlib.` part for me.

[^1]: still, that could be one use of `sed` to replace everything, but let's say that this would be ugly.

Reply:
I agree, let's just call e.g. `stdlib.add-int128` directly.

Reply:
Changed for hardcoding the prefix on the function names.